index.html

```diff
@@ -90,7 +97,7 @@ <h2 id="bios-headind">
                     <p id="bio1-description" class="bio-box__description justified">
                         After he had disrupted the marble counter-top industry, Alphonso
                         became co-founder and main inflatable men engineer at
-                        ZanyMen.
+                        ZanyMen. <span></span>
```

Comment:
I'm not sure what these are for?

I also don't see any other change to this section but the intro now covers the header for the Bio section when you view it on mobile.